deploy/crds/crd-orders.v1beta1.yaml

```diff
@@ -102,6 +102,9 @@ spec:
                     name:
                       description: Name of the resource being referred to.
                       type: string
+                notAfter:
+                  description: Duration is the duration for the not after date for the requested certificate. this is set on order creation as pe the ACME spec.
```

Comment:
Field is named `notAfter` but comment talks about `duration` - what's the intention here in terms of naming? (from Certificate->CertificatreRequest->Order)

Reply:
should have been renamed.... looking into it

Reply:
fixed overlooked the json tag